### PR TITLE
Update and rename add-network.md to wallet-api.md

### PR DESCRIPTION
**Pull request by abdullahvenly**
Added Wallet-API section. I will also add it to menu in this same PR.

### CODE REVIEW

**Comment by abdullahvenly** [comment]
Hi @kmurphypolygon 

Added a new guide and also added it to the menu.

File: docs/tools/wallets/venly/wallet-api.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+| Blockchain interactions     | Developers can integrate functionalities like reading data from the blockchain or writing data to it, more commonly known as creating and interacting with Smart Contracts. |
+| Security features           | The API might offer features to enhance the security of user funds and transactions.                                                                                        |
+| User experience enhancement | It can contribute to a smoother and more user-friendly interaction with blockchain applications.                                                                            |
+| Multi-blockchain support    | Venly supports multiple blockchain networks, allowing developers to offer wallets for different cryptocurrencies. |
+
+## Prerequisites
+
+1. You need a Venly business account, if you don't have one, click [here](https://portal.venly.io) to register in our Developer Portal, or follow our step-by-step guide, [Getting Started with Venly](https://venly.readme.io/docs/getting-started).
```
Comment (kmurphypolygon) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
1. You need a Venly business account. If you don't have one, click to register in our [Developer Portal](https://portal.venly.io), or follow our [Getting Started with Venly](https://venly.readme.io/docs/getting-started) guide.
```

File: docs/tools/wallets/venly/wallet-api.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+| :--------------- | :--------- | :--------- | :-------------------------------------------------------------------------------------------- | :-------------------------------------------- |
+| `Signing-Method` | Header     | `id:value` | `id`: This is the ID of the signing method. `value`: This is the value of the signing method. | `756ae7a7-3713-43ee-9936-0dff50306488:123456` |
+
+| Parameter                | Param type | Description                                            | Data type | Mandatory |
+| :----------------------- | :--------- | :----------------------------------------------------- | :-------- | :-------- |
+| `secretType`             | Body       | The blockchain on which to create the wallet           | String    | ✅         |
+| `userId`                 | Body       | The ID of the user who you want to link this wallet to | String    | ❌         |
+| `pincode` (_Deprecated_) | Body       | The pin that will encrypt and decrypt the wallet       | String    | ❌         |
+
+### Request Body
```
Comment (kmurphypolygon) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
### Request body
```

File: docs/tools/wallets/venly/wallet-api.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+```json
+{
+  "secretType": "MATIC",
+  "userId": "6a5a9020-e969-4d9a-ae4b-fcd91a75769d"
+}
+```
+
+### Response Body
```
Comment (kmurphypolygon) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
### Response body
```

File: docs/tools/wallets/venly/wallet-api.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            "rawBalance": "0",
+            "rawGasBalance": "0",
+            "decimals": 18
+        }
+    }
+}
+```
+
+!!! success "Ready to try it out?"
+    [Click here to read the getting started guide for Wallet-API.](https://docs.venly.io/docs/wallet-api-getting-started)
```
Comment (kmurphypolygon) [inline docs/tools/wallets/venly/wallet-api.md]:
```suggestion
    Click to read the [getting started guide for Wallet API.](https://docs.venly.io/docs/wallet-api-getting-started)
```

File: mkdocs.yml [hunk 5]
```diff
@@ -474,7 +474,7 @@ nav:
           - Venly:
               - Intro:  tools/wallets/venly/index.md
               - Create wallet:  tools/wallets/venly/create-wallet.md
-              - Add network:  tools/wallets/venly/add-network.md
+              - Wallet-API:  tools/wallets/venly/wallet-api.md
```
Comment (kmurphypolygon) [inline mkdocs.yml]:
```suggestion
              - Wallet API:  tools/wallets/venly/wallet-api.md
```

**Review by kmurphypolygon — APPROVED** [review]
LGTM

**Comment by kmurphypolygon** [comment]
Hi @abdullahvenly thank you for your contribution. Check to see your content live in the next day or so.

**Comment by kmurphypolygon** [comment]
Hi @abdullahvenly we were checking your PR as I'm just wondering why you have chosen to use an external CDN rather than our repo for your images?

I have reverted the PR for this reason. 

Please go ahead and raise it again but could you add your images to our img/ folder in the correct subfolder. Thank you.

**Pull request merged**